pkg/core/config.go

```diff
@@ -347,14 +335,35 @@ func NewConfiguration(path string) (cfg Configuration) {
 		os.Exit(1)
 	}
 
+	//load the policy file
+	policyEnforcer, err := loadPolicyFile(util.EnvOrDefault("LIMES_API_POLICY_PATH", "/etc/limes/policy.yaml"))
+	if err != nil {
+		logg.Fatal(err.Error())
+	}
+	apiCfg := APIConfiguration{
+		ListenAddress:  util.EnvOrDefault("LIMES_API_LISTEN_ADDRESS", ":80"),
+		PolicyEnforcer: policyEnforcer,
+	}
+	allowedOrigins := strings.ReplaceAll(os.Getenv("LIMES_API_CORS_ALLOWED_ORIGINS"), " ", "")
+	apiCfg.CORS.AllowedOrigins = strings.Split(allowedOrigins, "||")
+	exceptCodeStrings := strings.Split(os.Getenv("LIMES_API_REQUEST_LOG_EXCEPT_STATUS_CODES"), ",")
+	exceptCodes := make([]int, 0, len(exceptCodeStrings))
+	for _, v := range exceptCodeStrings {
+		v := strings.TrimSpace(v)
+		code, err := strconv.Atoi(v)
+		if err != nil {
+			logg.Fatal(err.Error())
```

Comment:
The error message should indicate where the parse error occurred:
```suggestion
			logg.Fatal("while parsing LIMES_API_REQUEST_LOG_EXCEPT_STATUS_CODES: %s", err.Error())
```